tensorflow/python/ops/array_ops.py

```diff
@@ -322,14 +322,14 @@ def setdiff1d(x, y, index_dtype=dtypes.int32, name=None):
   out[i] = x[idx[i]] for i in [0, 1, ..., len(out) - 1]
   ```
 
-  #### EXAMPLE:
+  Example usage:
 
   >>> x = [1, 2, 3, 4, 5, 6]
   >>> y = [1, 3, 5]
-  >>> result = tf.setdiff1d(x,y)
-  >>> sess = tf.compat.v1.Session()
-  >>> print(sess.run(result))
-  ListDiff(out=array([2, 4, 6], dtype=int32), idx=array([1, 3, 5], dtype=int32))
+  >>> result = setdiff1d(x,y)
+  >>> ListDiff(out=<tf.Tensor: id=76, shape=(3,), dtype=int32, numpy=
```

Comment:
I think you want to just write setdiff1d(x, y) instead of result = to make the doctest pass; similarly remove >>> from the result line and ... from subsequent result lines

Reply:
Thanks @alextp.